go.mod

```diff
@@ -26,6 +26,18 @@ replace (
 	gopkg.in/yaml.v3 => github.com/unix4ever/yaml v0.0.0-20220527175918-f17b0f05cf2c
 )
 
+// fd-leak related replacements: https://github.com/siderolabs/talos/issues/9412
```

Comment:
Maybe leave a TODO/FIXME to not forget to replace these versions with upstream ones once the fixes are upstreamed. Maybe we can just leave the issue open instead

Reply:
that's what I'm doing: 

> I'll keep the issue open to track upstream PR status and remove replace directives.